### PR TITLE
build(root): lint commit messages

### LINKED ISSUE #147: lint commit messages
Enforce conventional commits since we rely on them for generating release changelogs.

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #147

### Summary

Using https://commitlint.js.org/.

I enforced that a scope is present in the message. If we notice that it feels too restrictive, we can remove that check later.

The last dependabot commit broke the `body-max-line-length` default of 100, so I set it to 200.

### PR Checklist

- ~~[ ] The implemented feature is covered by an appropriate test.~~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "dashboards",
+  "private": true,
+  "version": "0.0.1",
+  "description": "A collection of GenSpectrum dashboards",
```
Comment (JonasKellerer) [inline package.json:5]: Is this the right description? It seems to be the dashboards as well as the components.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "type": "module",
+  "devDependencies": {
+    "@commitlint/cli": "^19.2.2",
+    "@commitlint/config-conventional": "^19.2.2",
+    "@commitlint/types": "^19.0.3"
+  }
+}
```
Comment (JonasKellerer) [inline package.json:12]: Not this PR, but maybe we move our `prettierrc.json` also to top level. Then, intelliJ finds it in the example, as well as in the components folder, I think.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "name": "dashboards",
+  "private": true,
+  "version": "0.0.1",
+  "description": "A collection of GenSpectrum dashboards",
+  "type": "module",
+  "devDependencies": {
+    "@commitlint/cli": "^19.2.2",
+    "@commitlint/config-conventional": "^19.2.2",
+    "@commitlint/types": "^19.0.3"
+  }
```
Comment (JonasKellerer) [inline package.json:11]: Do we want to have a short action to run also locally? Or is this not possible?

File: package.json (hunk 2 above)
Comment (fengelniederhammer) [reply package.json:12]: I also thought about that. Let's do it in a later PR :+1:

**Pull request merged**